src/ECER.Managers.Registry.Contract/Applications/Contract.cs

```diff
@@ -201,3 +219,17 @@ public record SubmitReferenceCommand(string Token) : IRequest<ReferenceSubmissio
   public WorkExperienceReferenceSubmissionRequest? WorkExperienceReferenceSubmissionRequest { get; set; }
   public CharacterReferenceSubmissionRequest? CharacterReferenceSubmissionRequest { get; set; }
 }
+
+public record SubmittedApplicationStatus(string Id, DateTime SubmittedOn)
+{
```

Comment:
I'm not sure why these are included in an application status response - if this is the entire application, then it's not a status